project/Dependencies.scala

```diff
@@ -35,7 +35,7 @@ object Dependencies {
     val AkkaPersistenceCassandra = "1.3.0"
     val AkkaPersistenceJdbc = "5.5.0"
 
-    val AkkaPersistenceR2dbc = "1.3.0"
+    val AkkaPersistenceR2dbc = "1.3.0+11-b3f1fca7-SNAPSHOT"
```

Comment:
This seems off?

Reply:
thanks, reverted
I'll bump this to 1.3.1 before releasing but that is another pr